.github/workflows/ci-linux-artifacts.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           path: |
             avifenc
             avifdec
+            avifgainmaputil
```

Comment:
In ci-windows-artifacts.yml, the `path` field is specifed as:

```
          path: "*.exe"
```

So avifgainmaputil.exe is included in the artifacts.